addons/mail/static/src/components/composer/composer.scss

```diff
@@ -231,8 +245,11 @@
 
 .o_Composer_coreMain:not(.o-composer-is-compact) {
     background: white;
-    border-radius: 5px;
     border: 1px solid lighten(gray('400'), 5%);
+
+    &:not(.o-composer-is-extended) {
+        border-radius: 5px;
+    }
```

Comment:
That is the reason for changing this rule in that case?

Reply:
Because the textarea should not be rounded when there is a subject as the subject is on top of it and they should be "glued" to each other

Reply:
Ok, worth commenting this line in css file.